manifests/dynflow/worker.pp

```diff
@@ -47,7 +47,7 @@
       content => foreman::to_symbolized_yaml($config),
     }
     ~> service { $service:
-      ensure    => running,
+      ensure    => true,
```

Comment:
What does `ensure => true` mean?

https://www.puppet.com/docs/puppet/8/types/service.html#service-attribute-ensure is not exactly verbose on that.

Reply:
According to https://github.com/puppetlabs/puppet/blob/3192f4aab419b221402f738ed1be90de0d969be5/lib/puppet/type/service.rb#L114 it should not be a change at all?